tests/testCmdLineTask.py

```diff
@@ -28,14 +30,14 @@
 import lsst.utils
 import lsst.pex.logging as pexLog
 import lsst.pipe.base as pipeBase
-from lsst.obs.test import TestConfig
+import lsst.obs.test
```

Comment:
@PaulPrice you need to do this to stop pytest auto-discovering a class called `Test*` in the module namespace and deciding that it's a test class.